docs/source/developer_zone/build_locally.rst

```diff
@@ -323,8 +336,21 @@ or with ``libmamba``:
 Tests
 *****
 
-You should be able to run the Python-based test suite:
+In order to run the Python-based test suite, you need to set the following environment variables:
+
+.. code::
+
+    export TEST_MAMBA_EXE=build/micromamba/micromamba
+    export MAMBA_ROOT_PREFIX=YOUR_MICROMAMBA_ROOT_PREFIX
+
+Then, from the ``mamba`` root directory, you should be able to run the tests:
 
 .. code::
+    cd ..
```

Comment:
Which folder did we start from? Should we simply remove this?

Reply:
We were in the build to run `make`, so I added this eventually so that it would be consistent with the previous steps and also added the  `Then, from the ``mamba`` root directory, ...` just before so that it's clearer...

Reply:
Two folders are named `mamba`, the repo and the sub-folder.

I you care to change that, I think using `cmake -B build/ -D ...` & `cmake --build build/` above, then removing all `cd`/`mkdir` would be nicer.

Reply:
Done!